CHANGELOG.md

```diff
@@ -2,13 +2,14 @@
 
 ## Unreleased
 
-## 1.3 - 2015-07-03
+## 1.3 - 2015-07-27
```

Comment:
@ruslankhaertdinov no need to change release date here, just add your new line in the "Unreleased" section.